packages/ni.measurementlink.discovery.v1/pyproject.toml

```diff
@@ -0,0 +1,98 @@
+[tool.poetry]
+name = "ni.measurementlink.discovery.v1"
```

Comment:
This package name is incorrect. It should end with ".proto". Rename the directory too.

Add a step to one of the check_*.yml workflows to enforce this. It can be something like
```
if: ${{ !endsWith(inputs.package-name, '.proto') && inputs.package-name != 'ni.protobuf.types' }}
run: echo "::error title=Package Name Error::Invalid package name ${{ inputs.package-name }}. The package name must end in '.proto'."
```

Reply:
This seems oddly specific check to add here. As we add more message only proto files we would need to update this. Is there any way to check whether this is a versioned service proto or a message-only proto?

Reply:
On second thought, XOR would be more appropriate because we don't want both.
```
if: ${{ (!startsWith(inputs.package-name, 'ni.protobuf.') && !endsWith(inputs.package-name, '.proto')) || ( startsWith(inputs.package-name, 'ni.protobuf.') && endsWith(inputs.package-name, '.proto')) }}
run: echo "::error title=Package Name Error::Invalid package name ${{ inputs.package-name }}. The package name must start with `ni.protobuf.` or end with '.proto', but not both."